link_shortener/view_routes.py

```diff
@@ -68,7 +69,15 @@ async def about_page(request):
 @login_required
 @credential_whitelist_check
 async def all_active_links(request, user):
-    link_data = await retrieve_links(request, filters={'is_active': True})
+    filter_set = {'is_active', 'owner', 'search'}
+    filters = {element: request.args.get(element) for element in filter_set}
+
+    if filters['is_active'] is None:
+        filters['is_active'] = True
```

Comment:
Tuhle logiku bych možná přehodil do filter_links

Reply:
do filter_links nemoze, lebo na zaklade toho ziadame DB o zakladny set dat - active inactive. Pak robime nas filtering.
co dostaneme pre active/inactive z FE je is_active=false, alebo nic. defaultne mame vratit vsetok active.

zmanil som to tak:
721b4bbf8c3cbc029d35eac0937ab598f74669a0